src/main/java/order/controller/OrderController.java

```diff
@@ -0,0 +1,85 @@
+package order.controller;
+
+import java.util.List;
+import order.domain.Order;
+import order.domain.OrderResult;
+import order.domain.Orders;
+import order.domain.ServiceMenu;
+import order.domain.dto.OrderResponse;
+import order.domain.dto.ServiceResponse;
+import order.global.util.OrderInputParser;
+import order.view.InputView;
+import order.view.OutputView;
+
+public class OrderController {
+    private final InputView inputView;
+    private final OutputView outputView;
+
+    public OrderController(InputView inputView, OutputView outputView) {
+        this.inputView = inputView;
+        this.outputView = outputView;
+    }
+
+    public void run() {
+        Orders orders = generateOrders();
+        OrderResult orderResult = generateOrderResult(orders);
+        ServiceMenu serviceMenu = generateServiceMenu(orders);
+
+        displayOrderSummary(orders, orderResult, serviceMenu);
+
+        inputView.closeScanner();
+    }
+
+    private void displayOrderSummary(Orders orders, OrderResult orderResult, ServiceMenu serviceMenu) {
+        displayOrderDetails(orders);
+        displayTotalAmount(orderResult);
+        displayDeliveryFee(orderResult);
+        displayServiceMenu(serviceMenu);
+        displayFinalAmount(orderResult);
+    }
+
+    private void displayFinalAmount(OrderResult orderResult) {
+        int finalAmount = orderResult.calculateFinalAmount();
+        outputView.printFinalAmount(finalAmount);
+    }
+
+    private void displayServiceMenu(ServiceMenu orderService) {
+        ServiceResponse response = orderService.createResponse();
+
+        if (orderService.hasServiceQuantity()) {
+            outputView.printServiceMenu(response);
+        }
```

Comment:
```suggestion
if (orderService.hasServiceQuantity()) {
    outputView.printServiceMenu(orderService.createResponse());
}
```
여기도 마찬가지로 메서드명을 통해서 그리 어렵지 않게 response를 불러온다는 것을 알 수 있을 것 같습니다

Reply:
이 아래의 메서드들도 이런식으로 변경할 수 있을 것 같습니다

Reply:
짚어주셔서 감사합니다!! 앞으로 최대한 적용해보겠습니다!